server/api/api.go

```diff
@@ -77,6 +77,13 @@ func (a *API) RegisterRoutes(r *mux.Router) {
 	apiv2.Use(a.panicHandler)
 	apiv2.Use(a.requireCSRFToken)
 
+	/*
```

Comment:
Opinionated nit here, but can we add a `ToDo:` or some notation like that to mark this as an actionable comment?

```suggestion
	/* ToDo:
```